modules/ROOT/nav.adoc

```diff
@@ -1,3 +1,63 @@
+* Workflows

```

Comment:
We have an about page for this one (Heena has a gdoc, we can add that page directly here). Also, the current overview page (link in the Documentation menu) must be present.

Reply:
I added the overview in sidebar nav. Will check with Heena about the "About" section.